### PR TITLE
Update install-kubeadm.md

### PR DESCRIPTION
**Pull request by yfge**
hello ,guys 

I think I made some mistakes on reading the doc 
so I change it a little
Thanks

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ ARCH="amd64"
 curl -L "https://github.com/kubernetes-sigs/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz" | sudo tar -C $DOWNLOAD_DIR -xz
 ```
 
-Install `kubeadm`, `kubelet`, `kubectl` and add a `kubelet` systemd service:
+Install `kubeadm`, `kubelet` and add a `kubelet` systemd service:
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:246]: line 261 tells how to install kubectl separately, so the old list of components seems correct here.

Comment (yfge) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:246]: well I think the bash scripts started at line 248  is supporting line 246, line 261 is the next step .
the line 246 make ones to think kubectl has been installed through the scripts.

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:246]: i think you are making a good point. let's leave it to the pr approvers to decide.

Comment (tengqm) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:246]: I think the current description is acceptable.
The operations that follow this line include the installation of the binaries and the creation of the systemd service.
You can still challenge that why start the `kubelet` service after downloading `kubectl`. Well, sometimes the order doesn't matter that much. The irrelevance is implied in the operations themselves.

**Comment by kbhawkey** [comment]
@yfge . The original text seems okay. Do you agree?

**Comment by natalisucks** [comment]
@yfge I also agree that the original text should stay and is relevant, so I don't think we can accept your changes.

However, we're still happy that you're choosing to contribute to the Kubernetes docs, so I'd like to encourage you to look through some of our [triaged issues](https://github.com/kubernetes/website/labels/triage%2Faccepted) in order to make another contribution. Thank you!

With the above being said, I am closing this PR.

/close

**Pull request closed without merging**